test/Makefile

```diff
@@ -30,6 +30,8 @@ test:
 	  || git -C code clone ../../../ocamlformat
 	test -d code/infer \
 	  || git -C code clone "https://github.com/facebook/infer.git"
+	test -d code/ocaml \
+	  || git -C code clone "https://github.com/hhugo/ocaml.git"
```

Comment:
Why not take the official one git repo ?

Reply:
I didn't know if you had made some additional changes wrt the official one.  I'm happy to change it.

Reply:
I've formatted the code using -no-comment-check to get a larger portion of the codebase formatted.

Reply:
I tried that, and realized that no-comment-check isn't parsed in config files

Reply:
This was on purpose. I didn't want that behavior to trigger without the flag appearing in the cmd line.